core/src/main/java/org/opensearch/sql/storage/read/TableScanBuilder.java

```diff
@@ -109,6 +109,7 @@ public <R, C> R accept(LogicalPlanNodeVisitor<R, C> visitor, C context) {
     return visitor.visitTableScanBuilder(this, context);
   }
 
-  public void pushDownOffset(int i) {
+  public boolean pushDownOffset(int i) {
+    return false;
```

Comment:
Will be removed, keep unchanged
```suggestion
  public void pushDownOffset(int i) {
```